browser/lib/getTodoState.js

```diff
@@ -0,0 +1,25 @@
+export function getTodoState (content) {
```

Comment:
How do you think about `getTodoState.js` file name? Should change to `getTodoStatus.js`?